src/hokusai-app-updater/update_apps.sh

```diff
@@ -52,7 +52,8 @@ function commit() {
   fi
 
   echo "### commit changes ###"
-  git commit -am "$MSG" --no-verify
+  git add .
+  git commit -m "$MSG" --no-verify
```

Comment:
`git commit -am` doesn't add untracked files/directories. Since this change adds a new directory, I had to modify this step.